_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -27,107 +27,141 @@ required_by:
+A RelationshipTemplate is generally used to establish a Relationship between two Identities. A Request can be sent in this process of establishing a Relationship.
+Nevertheless, a RelationshipTemplate can also be used to exchange Requests between Identities that have already established a Relationship.
+For more information on how to establish Relationships, refer to the [Establish Relationships]({% link _docs_integrate/establish-relationships.md %}) scenario documentation.
+{: .notice--info}
 
-## Check your Request's validity
+## Check the Request's validity
 
-Firstly, you should [check if your Request is valid]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}).
-As an example, we use a Request with just an [AuthenticationRequestItem]({% link _docs_integrate/data-model-overview.md %}#authenticationrequestitem), but you can use any Request you want.
-For an overview of the available [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitems), check out our [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
+The Sender wants to construct a Request that it can send to the Recipient by inserting it into a RelationshipTemplate.
+Before the Request is sent via the RelationshipTemplate, the Sender should first check its validity by proceeding as described in the documentation of the [Check if outgoing Request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}) use case.
+To do this, the Request must be specified in the `content` property of the payload.
+As an example in this guide, a [Request]({% link _docs_integrate/data-model-overview.md %}#request) containing only a single [AuthenticationRequestItem]({% link _docs_integrate/data-model-overview.md %}#authenticationrequestitem) within its `items` property is used, but any other Request can also be used.
 
-```json
+```jsonc
 {
   "content": {
+    // Specification of a Request
+    "title": "Example of a Request",
     "items": [
       {
         "@type": "AuthenticationRequestItem",
         "mustBeAccepted": true,
-        "title": "The Sender is asking for an authentication."
+        "title": "The Sender is asking for an authentication"
       }
     ]
   }
 }
 ```
 
-Even though the Requests are validated during the RelationshipTemplate creation you should not skip this step as it gives you additional information in case of validation errors.
+If a Request is contained within a RelationshipTemplate, its validity is automatically checked when the [RelationshipTemplate is created](#create-the-relationshiptemplate). The RelationshipTemplate cannot be created if the Request is faulty.
+Nevertheless, the Request's validity should be checked before attempting to create the RelationshipTemplate in order to obtain additional information about the reasons for the error in the case of a faulty Request.
```

Comment:
According to my research, there are both the phrases "in case of" and "in the case of" and in this particular context "in the case of" would be more appropriate. A comma is used with "in order to" if the phrase is at the beginning of the sentence, otherwise it is not needed.